tests/integration_tests/build/test_binary_size.py

```diff
@@ -44,7 +44,13 @@
 
 @pytest.mark.timeout(500)
 def test_binary_sizes():
-    """Test if the sizes of the release binaries are within expected ranges."""
+    """
+    Test if the sizes of the release binaries are within expected ranges.
+
+    @type: integration
```

Comment:
What are the possible values for this? Integration and performance?

Reply:
We could go down two different paths here: have a list of preset values (e.g.  `integration`, `performance`, `regression`) or add anything that we want.

WDYT? The first option sounds a bit better I think.

Reply:
Yes, the first option sounds better. How much effort would it take to implement it? Do any of the fields (criteria, issue, name) have any validation in place?

Reply:
No, we're currently just picking up the fields without any validation.

It's pretty easy to implement I think. I'll take a look and circle back to this PR :)